README.md

```diff
@@ -947,6 +949,97 @@ A simple use-case could be as follows:
+| `message`     | The body of the notification                        | String                          | Required  |
+| `title`       | The title of the notification                       | String                          | Allowed   |
+| `subscribers` | The list of subscribers to send the notification to | Subscriber elements (see below) |  Required |
+
+The `subscribers` element can contain multiple third party subscribers and/or Yudu subscribers, but must contain at minimum one of either.
+
+| Subscriber Element          | Description                       | Type   |
+| --------------------------- | --------------------------------- | ------ |
+| `thirdPartySubscriberToken` | Third party subscriber identifier | String |
+| `subscriberUsername`        | Yudu subscriber username          | String |
+
+
+#### Supported Verbs
+| URI                      | Relation                                      | Verbs |
+| ------------------------ | --------------------------------------------- | ----- |
+| `/targetedNotifications` | `http://schema.yudu.com/targetedNotification` | POST  |
+
+A **POST** request sends a targeted notification to the specified list of subscribers, and returns an XML representation of the response.
+
+#### Targeted Notification Response
+A targeted notification response will be returned as an XML representation, providing information about the success of iOS push notifications and Firebase cloud messages, and reasons for failures.
+
+##### XML Failure Response
+``` xml
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<targetedNotificationResponse xmlns="http://schema.yudu.com">
+    <firebaseResponse>
+        <allSucceeded>false</allSucceeded>
+        <responseMessage>
+            Firebase certificate details are missing for this node.
+            Please ensure the Firebase private key has been uploaded and selected,
+            and the project ID supplied on the "editPublication.htm" page.
+        </responseMessage>
+    </firebaseResponse>
+    <iOSResponse>
+        <allSucceeded>false</allSucceeded>
+        <responseMessage>
+            Push notification certificate details are missing for this node.
+            Please ensure the APNS security certificate has been uploaded and selected,
+            and the password supplied on the "editPublication.htm" page.
+        </responseMessage>
+    </iOSResponse>
+</targetedNotificationResponse>
+```
+
+##### XML Success Response
+``` xml
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<targetedNotificationResponse xmlns="http://schema.yudu.com">
+    <firebaseResponse>
+        <allSucceeded>true</allSucceeded>
+        <responseMessage>
+            Successfully completed sending Firebase messages.
+            There were 1 messages to send, and 1 sent successfully.
+        </responseMessage>
+    </firebaseResponse>
+    <iOSResponse>
+        <allSucceeded>false</allSucceeded>
```

Comment:
Surely this should be `true`.

Reply:
Yes it should. Thanks for pointing this out.